Movement.h

```diff
@@ -10,27 +10,29 @@
 #include "Control.h"
 #include "DropKit.h"
 #include "arduino.h"
+#include "BNO.h"
 
 class Movement {
     
   public:
     Movement();
     Control control;
     SensorMap sensor;
+    BNO logic;
```

Comment:
I think you're gonna have issues if you declare the BNO object in several places. 
Either decide to use it only in one file, or send a pointer to the object in the constructor.

Reply:
All sensors must be initialized only once, otherwise there are gonna be issues.